### PR TITLE
[Twig Bridge] Show errors in expanded choice widget.

### PR DESCRIPTION
**Pull request by Warxcell**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | I think no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | no
| License       | MIT
| Doc PR        | no

I have several choices - I want to attach error to specific choice - but it's not shown.
Let's say that specific code is executed behind every choice - I want to show error at that specific choice - because that's error behind the chosen option - not the whole choice field.

screen shot 2017-11-23 at 2 48 29 pm

### CODE REVIEW

**Comment by Warxcell** [comment]
Added `has-error` to specific radio button.

screen shot 2017-11-23 at 3 01 49 pm


If there are no errors attached to specific radio button - Everything should be working as before.

File: src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig [hunk 1]
```diff
@@ -107,6 +107,7 @@
                     parent_label_class: label_attr.class|default(''),
                     translation_domain: choice_translation_domain,
                 }) -}}
+                {{ form_errors(child) }}
```
Comment (nicolas-grekas) [inline src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: missing dash right side, isn't it? `{{ form_errors(child) -}}`

Comment (nicolas-grekas) [reply src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig]: ping @Warxcell

**Comment by nicolas-grekas** [comment]
Closing due to inactivity, feel free to resubmit.

**Pull request closed without merging**